app/src/main/java/com/oneshot/rxjavaapp/presentation/presenter/BasePresenter.java

```diff
@@ -0,0 +1,39 @@
+package com.oneshot.rxjavaapp.presentation.presenter;
+
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.disposables.Disposable;
+
+/**
+ * Created by brettn on 4/25/17.
+ */
+
+public class BasePresenter implements PresenterInterface {
+    private CompositeDisposable mDisposables = new CompositeDisposable();
```

Comment:
Does this replace Subscriptions and we no longer need to do subscribe/unsubscribe ?